enable-vcp-image/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:16.10
```

Comment:
We cannot put this kind of scripts in the vsphere package on Kubernetes Repository. It is restricted to Cloud Provider code and Volume plug-in code.